shared/src/userData.ts

```diff
@@ -30,6 +30,7 @@ export interface Business {
   readonly formationData: FormationData;
   readonly environmentData: EnvironmentData | undefined;
   readonly version: number;
+  readonly userId: string;
```

Comment:
Just kindof a musing... How would y'all think about making this into an array value? This could potentially allow us to have a 1:many relationship between business:user. Maybe a little bit of YAGNI, but I could see this being a nice support feature in the future even if we don't want to work on building user permissions into the system at this present time.

@aadedejifearless @seidior